src/app.rs

```diff
@@ -701,7 +701,34 @@ where
     ) -> u64 {
         self.init_modules(|router, _, _| router.wasm.store_code(creator, code))
     }
+}
 
+// Helper functions to call some custom WasmKeeper logic.
+// They show how we can easily add such calls to other custom keepers (CustomT, StakingT, etc)
+impl<BankT, ApiT, StorageT, CustomT, StakingT, DistrT, IbcT, GovT>
+    App<
+        BankT,
+        ApiT,
+        StorageT,
+        CustomT,
+        WasmKeeper<CustomT::ExecT, CustomT::QueryT>,
+        StakingT,
+        DistrT,
+        IbcT,
+        GovT,
+    >
+where
+    BankT: Bank,
+    ApiT: Api,
+    StorageT: Storage,
+    CustomT: Module,
+    StakingT: Staking,
+    DistrT: Distribution,
+    IbcT: Ibc,
+    GovT: Gov,
+    CustomT::ExecT: Clone + Debug + PartialEq + JsonSchema + DeserializeOwned + 'static,
+    CustomT::QueryT: CustomQuery + DeserializeOwned + 'static,
+{
```

Comment:
This code is newer on origin/main. I guess refreshing/updating your fork should help.

Reply:
Okay, I'll do a rebase and compare the changes.